src/opam.mli

```diff
@@ -101,3 +103,6 @@ val exec : t -> Switch.t -> args:string list -> Cmd.t
 
 (** Check that two instances of [Opam] are equal. *)
 val equal : t -> t -> bool
+
+(** Check that a package is installed in the given switch. *)
+val has_package : t -> Switch.t -> string -> bool Promise.t
```

Comment:
naked string arugments should be labelled. We might also have a type for package names.